examples/kitchensink-ts/config/admin.ts

```diff
@@ -1,4 +1,6 @@
-export default ({ env }) => ({
+import type { Config } from '@strapi/strapi';
+
+const adminConfig: Config.Admin = ({ env }) => ({
```

Comment:
Did you add this just for testing ? `Config` type ?
if we want the configs to looks like this we probably need to change the template of the generated configs

Reply:
I didn't want to change the templates until we were sure about what we were doing but yeah, this is more like a test/demo of how it would work. I'm assuming this will be replaced with a factory call in the end before we merge the feature.